packages/dx-react-scheduler/src/plugins/appointment-form.jsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import { getMessagesFormatter } from '@devexpress/dx-core';
+import {
+  Plugin,
+  Action,
+  createStateHelper,
+} from '@devexpress/dx-react-core';
+import {
+  setAppointment,
+  COMMIT_COMMAND_BUTTON,
+  CANCEL_COMMAND_BUTTON,
+} from '@devexpress/dx-scheduler-core';
+
+const defaultMessages = {
+  allDayText: 'All Day',
+  titleLabel: 'Subject',
```

Comment:
We use 'Subject' as a default value for a title)) It looks a bit strange. Can we change it to `titleLabel: 'Title'` ?

Reply:
I'm not against)